Header header
Point2D center
Size2D size
int32 orientation
